docker-compose.yml

```diff
@@ -0,0 +1,65 @@
+name: split-pro-prod
```

Comment:
same we already have a file under docker folder

Reply:
Good catch, will remove it!

Reply:
https://github.com/oss-apps/split-pro/pull/132/commits/a1caeed1f7cdf5c4546b2ff8868fa528abf0f781